src/cc/bpf_module_rw_engine.cc

```diff
@@ -416,7 +416,11 @@ int BPFModule::annotate() {
       StructType *st = dyn_cast<StructType>(t);
 #else
     if (PointerType *pt = dyn_cast<PointerType>(gvar->getType())) {
+# if LLVM_MAJOR_VERSION >= 14
```

Comment:
The simplest fix for this would be change the above `LLVM_MAJOR_VERSION >= 15` to `LLVM_MAJOR_VERSION >= 14`.